[2023-10-16T17:22:45.833+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup.TaskC scheduled__2023-10-15T00:00:00+00:00 [queued]>
[2023-10-16T17:22:45.835+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup.TaskC scheduled__2023-10-15T00:00:00+00:00 [queued]>
[2023-10-16T17:22:45.835+0100] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-16T17:22:45.839+0100] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): TaskC> on 2023-10-15 00:00:00+00:00
[2023-10-16T17:22:45.840+0100] {standard_task_runner.py:57} INFO - Started process 17289 to run task
[2023-10-16T17:22:45.843+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup', 'TaskC', 'scheduled__2023-10-15T00:00:00+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/catchup.py', '--cfg-path', '/var/folders/_3/bj8r7kz51nz4g7hl_dpr6jdw0000gn/T/tmpt9gps4p0']
[2023-10-16T17:22:45.844+0100] {standard_task_runner.py:85} INFO - Job 153: Subtask TaskC
[2023-10-16T17:22:45.865+0100] {task_command.py:416} INFO - Running <TaskInstance: cron_catchup.TaskC scheduled__2023-10-15T00:00:00+00:00 [running]> on host 192.168.1.112
[2023-10-16T17:22:45.888+0100] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='workflow' AIRFLOW_CTX_DAG_ID='cron_catchup' AIRFLOW_CTX_TASK_ID='TaskC' AIRFLOW_CTX_EXECUTION_DATE='2023-10-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-15T00:00:00+00:00'
[2023-10-16T17:22:45.889+0100] {logging_mixin.py:151} INFO - task C executed
[2023-10-16T17:22:45.889+0100] {python.py:194} INFO - Done. Returned value was: None
[2023-10-16T17:22:45.891+0100] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cron_catchup, task_id=TaskC, execution_date=20231015T000000, start_date=20231016T162245, end_date=20231016T162245
[2023-10-16T17:22:45.901+0100] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-16T17:22:45.908+0100] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-16T17:32:06.234+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup.TaskC scheduled__2023-10-15T00:00:00+00:00 [queued]>
[2023-10-16T17:32:06.236+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup.TaskC scheduled__2023-10-15T00:00:00+00:00 [queued]>
[2023-10-16T17:32:06.236+0100] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-16T17:32:06.240+0100] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): TaskC> on 2023-10-15 00:00:00+00:00
[2023-10-16T17:32:06.241+0100] {standard_task_runner.py:57} INFO - Started process 18496 to run task
[2023-10-16T17:32:06.248+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup', 'TaskC', 'scheduled__2023-10-15T00:00:00+00:00', '--job-id', '204', '--raw', '--subdir', 'DAGS_FOLDER/catchup.py', '--cfg-path', '/var/folders/_3/bj8r7kz51nz4g7hl_dpr6jdw0000gn/T/tmpygwg_80_']
[2023-10-16T17:32:06.257+0100] {standard_task_runner.py:85} INFO - Job 204: Subtask TaskC
[2023-10-16T17:32:06.275+0100] {task_command.py:416} INFO - Running <TaskInstance: cron_catchup.TaskC scheduled__2023-10-15T00:00:00+00:00 [running]> on host 192.168.1.112
[2023-10-16T17:32:06.294+0100] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='workflow' AIRFLOW_CTX_DAG_ID='cron_catchup' AIRFLOW_CTX_TASK_ID='TaskC' AIRFLOW_CTX_EXECUTION_DATE='2023-10-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-15T00:00:00+00:00'
[2023-10-16T17:32:06.294+0100] {logging_mixin.py:151} INFO - task C executed
[2023-10-16T17:32:06.294+0100] {python.py:194} INFO - Done. Returned value was: None
[2023-10-16T17:32:06.296+0100] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cron_catchup, task_id=TaskC, execution_date=20231015T000000, start_date=20231016T163206, end_date=20231016T163206
[2023-10-16T17:32:06.305+0100] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-16T17:32:06.311+0100] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
